ctl_main.c

```diff
@@ -1238,6 +1238,58 @@ static int cmd_showport(int argc, char *const *argv)
     return r;
 }
 
+static int cmd_showportparams(int argc, char *const *argv)
+{
+    int r = 0;
+
+    int br_index = get_index(argv[1], "bridge");
+    if(0 > br_index)
+        return br_index;
+
+    int i, count = argc - 3;
```

Comment:
if i read this correctly, if argc == 3, then count = 0 and nothing happens;
should we treat the `count == 0 ` case?

Reply:
This command should have [at least 3 argument](https://github.com/for454/mstpd/blob/777fc3e5630b96d301cf744f30c830372c73652b/ctl_main.c#L2257) that it means argc is at least equal to 4.

command syntax is like this:
```
 showportparams <bridge> <port> <param>... [params]
```